IconToggle/index.d.ts

```diff
@@ -2,10 +2,15 @@ import MaterialComponent from '../MaterialComponent';
 import { VNode } from 'preact';
 import { MDCFoundation, MDCComponent, MDCRipple } from '../MaterialComponentsWeb';
 
+declare interface IconToggleState {
```

Comment:
I'm not sure if we want to keep it consistent to prepend an `I` for interface like I've done for everything else.